### PR TITLE
feature(hydra-ssh): support gce on ssh/tunnel command

### PR DESCRIPTION
**Pull request by fruch**
with this change we can now the following commands:

```
hydra ssh --user `whoami`
hydra ssh-cmd rolling-upgrade--ubuntu-focal-db-node-6e9857ea-0-3 ls /
hydra tunnel --test-id 1234-1234-1234-1234
```

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@soyacz, can you take a quick look on this one ?

File: sct_ssh.py [hunk 1]
```diff
@@ -284,15 +391,32 @@ def copy_cmd(user, test_id, region, force_use_public_ip, src, dest):
...
+              help="User to search for (RunByUser tag)")
+@click.option("-t", "--test-id", default=None, help="test id to search for")
+def gcp_allow_public(user, test_id):
+    assert user or test_id
+    instances = select_instance_group(backends=['gce'], test_id=test_id, user=user)
+    for i in instances:
+        instances_client, info = get_gce_compute_instances_client()
+        gce_set_tags(instances_client=instances_client,
+                     instance=i,
+                     new_tags=['sct-allow-public'],
```
Comment (soyacz) [inline sct_ssh.py:419]: we could add some other tag for opening just ssh (e.g. sct-allow-public-ssh) which would open only ssh

**Review by soyacz — APPROVED** [review]
besides some general comments that could be converted to followup tasks
LGTM

File: sct_ssh.py [hunk 2]
```diff
@@ -284,15 +391,32 @@ def copy_cmd(user, test_id, region, force_use_public_ip, src, dest):
...
             group_id_to_add = aws_region.provide_sct_test_security_group(get_tags(i).get('TestId', 'N/A')).group_id
         all_sg_ids = list(set([sg['GroupId'] for sg in instance.security_groups] + [group_id_to_add]))
         instance.modify_attribute(Groups=all_sg_ids)
+
+
+@click.command("gce-allow-public", help="Attach test default security group to a group of instances")
+@click.option("-u", "--user", default=None,
+              help="User to search for (RunByUser tag)")
+@click.option("-t", "--test-id", default=None, help="test id to search for")
+def gcp_allow_public(user, test_id):
```
[the remark this answers is not in the document]
Comment (fruch) [reply sct_ssh.py:412]: I thought about it, but it was to give up one some of the functionall (which isn't actually used)
also it's not only for ssh, it's opening a whole lot more, also monitoring and everything configured

File: sct_ssh.py (hunk 1 above)
Comment (fruch) [reply sct_ssh.py:419]: it's not just for ssh, also for monitoring and such
I prefer it to use the `sct-allow-public` same as we do for test, yes we can split to to multiple tags, but again the functionally would be align to aws SGs (which we didn't separated per type)

so in general align those commands on both ends, need a bit extra work, which isn't needed for anything else right now

File: sct_ssh.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (fruch) [reply sct_ssh.py:412]: and those can be a followup tasks:
- align `gcp_allow_public` `attach_test_sg_cmd` hydra commands
- support Azure in `hydra ssh/tunnel` commands

**Pull request merged**